benchmark_utils/__init__.py

```diff
@@ -7,6 +7,5 @@
 from benchopt.utils import safe_import_context
 
 with safe_import_context() as import_ctx:
-    from .hugging_face_torch_dataset import HuggingFaceTorchDataset
-    from .image_dataset import ImageDataset
-    from .constants import constants
+    from .hugging_face_torch_dataset import HuggingFaceTorchDataset  # noqa: F401
+    from .image_dataset import ImageDataset  # noqa: F401
```

Comment:
To avoid relying on noqa, you can define what are the symbols which are exposed in this module with `__all__`.
```suggestion
    from .hugging_face_torch_dataset import HuggingFaceTorchDataset
    from .image_dataset import ImageDataset
    
    __all__ = ["HuggingFaceTorchDataset", "ImageDataset"]
```

But I am not sure it is worth it, I would remove these imports and directly import from the submodules in the dataset files.